packages/manager/modules/account-migration/src/notification/notification.html

```diff
@@ -0,0 +1,185 @@
+<oui-message
+    data-type="success"
+    data-ng-if="$ctrl.migrationDetail && !$ctrl.migrationDetail.isMigrationPending()"
+>
+    <span
+        data-translate="ovh_manager_account_migration_pending_contracts_success"
+    ></span>
+    <a
+        target="_blank"
+        class="oui-link_icon"
+        rel="noopener"
+        data-ng-href="{{:: $ctrl.faqLink }}"
+        data-ng-click="$ctrl.trackClick('faq')"
+    >
+        <span data-translate="ovh_manager_account_migration_faq_text"></span>
+        <span
+            class="oui-icon oui-icon-external-link"
+            aria-hidden="true"
+        ></span> </a
+    >.
```

Comment:
**nitpick**: The `.` could be in the translation

Reply:
The `.` should come after the icon. Hence we will not be able to make it a part of the translations I guess. Please let me know if you think otherwise. Thanks :)